package.json

```diff
@@ -48,6 +48,7 @@
     "util:copy-icons": "cpy \"./node_modules/@esri/calcite-ui-icons/js/*.json\" \"./src/components/icon/assets/icon/\" --flat",
     "util:deploy-next": "npm run util:prep-next && npm run util:push-tags && npm run util:publish-next",
     "util:deploy-next-from-ci": "ts-node --project ./tsconfig-node-scripts.json support/deployNextFromCI.ts",
+    "util:patch-es5-helpers": "ts-node support/patchES5Helpers.ts",
```

Comment:
can you add ` && npm run util:patch-es5-helpers` to the end of these release NPM scripts: 
- `release:prepare`
- `util:prep-next`

Reply:
I added it to `build` since all of the above reference it.